src/test/resources/GoogleSearch.feature

```diff
@@ -3,8 +3,15 @@ Feature: Search
   I want to search the internet
   So that results appropriate to my interests are returned
 
-  @FixMe
+  @SmokeTest
   Scenario: Relevant search results are returned to the user
     Given I am on the Google UK homepage
-    When I enter a search term
-    Then results relevant to the search term are returned
\ No newline at end of file
+    When I enter a search term 'BBC news'
```

Comment:
To parameterise this, we want to pass 'BBC new' in using double quotes - `"BBC News"` and then utilise that in your step def